morpheushelper/cogs/mod.py

```diff
@@ -260,11 +260,11 @@ async def report(self, ctx: Context, member: Member, *, reason: str):
     @commands.command()
     @Permission.warn.check
     @guild_only()
-    async def warn(self, ctx: Context, member: Member, *, reason: str):
+    async def warn(self, ctx: Context, member: Union[Member, int], *, reason: str):
```

Comment:
Please add `User` to the member type Union so that also Users that left the server can be used for the command.